Hardware/11-Wifi.md

```diff
@@ -25,3 +25,155 @@ Top Layer
 ![Wifi Top Layer](../assets/images/wifi_top.png)  
 Bottom Layer  
 ![Wifi Bottom Layer](../assets/images/wifi_bottom.png)  
+
+### Buttons
+
+The top push button, `RESET`, is a reset button that will power cycle the ESP8266 chip. Don't press the `RESET` button when a Ganglion is attached, if you want to power cycle the WiFi Shield, send a `;` command to the Ganglion or Cyton to trigger a power on reset of the WiFi Shield. The bottom button, `PROG`, is use for programming the WiFi Shield over serial UART and is hooked up to GPIO 0.
+
+#### Ganglion Note
```

Comment:
I think that everything below this line doesn't really belong on this tutorial. Looking at the Cyton and Ganglion it should only be specs and design files